### PR TITLE
[virt_autotest] reset unprivileged_userfaultfd value for guest migration

### PR DESCRIPTION
**Pull request by guoxuguang**
- Description:
Before kernel v5.11, The default value of unprivileged_userfaultfd sysctl is 1,
It means using the userfaultfd syscalls is allowed by unprivileged users.
Since kernel v5.11(current 5.14 for 15-SP4),  The default value of unprivileged_userfaultfd sysctl is
set to 0, In this case, The UFFD_USER_MODE_ONLY is needed if unprivileged
user(without SYS_CAP_PTRACE capability) want to use userfaultfd syscalls.
So, need to reset unprivileged_userfaultfd value on guest live migration dst - 15-SP4 KVM for our OSD guest migration test
- Related ticket: https://progress.opensuse.org/issues/100659
- Verification run: 
[virt-guest-migration-developing-from-developing-to-developing-kvm-dst](https://openqa.suse.de/tests/7527240#)
[virt-guest-migration-developing-from-developing-to-developing-kvm-src](https://openqa.suse.de/tests/7527241#)
[virt-guest-migration-sles12sp5-from-sles12sp5-to-developing-kvm-dst](http://openqa.suse.de/t7527246)
[virt-guest-migration-sles12sp5-from-sles12sp5-to-developing-kvm-src](http://openqa.suse.de/t7527247)
[virt-guest-migration-sles15sp3-from-sles15sp3-to-developing-kvm-dst](https://openqa.suse.de/tests/7531014#)
[virt-guest-migration-sles12sp5-from-sles12sp5-to-developing-kvm-src](https://openqa.suse.de/tests/7531015#)

### CODE REVIEW

**Comment by guoxuguang** [comment]
Add the latest verification run as below:
[virt-guest-migration-developing-from-developing-to-developing-kvm-dst](http://openqa.suse.de/tests/7563502#)
[virt-guest-migration-developing-from-developing-to-developing-kvm-src](http://openqa.suse.de/tests/7563503#)

File: tests/virt_autotest/guest_migration_dst.pm [hunk 1]
```diff
@@ -14,7 +14,17 @@ use testapi;
...
+use version_utils 'is_sle';
+
+#Since 15-SP4 kernel(v5.14), default value of unprivileged_userfaultfd sysctl is 0.
+#if unprivileged user want to use userfaultfd syscalls, we just only need to manually
+#reset unprivileged_userfaultfd value is 1 as the workaround on postcopy migration dst
+sub reset_unprivileged_userfaultfd {
+    script_run("sysctl -w vm.unprivileged_userfaultfd=1", 15);
+    script_run("sysctl -a | grep vm.unprivileged_userfaultfd", 15);
+    save_screenshot;
+}
```
Comment (waynechen55) [inline tests/virt_autotest/guest_migration_dst.pm:27]: Could you help move this function to the bottom of this file guest_migration_dst.pm ?

Comment (guoxuguang) [reply tests/virt_autotest/guest_migration_dst.pm]: sure, had moved it now.

**Comment by guoxuguang** [comment]
@alice-suse if there was not any new comment, please help merge this PR

**Pull request merged**